### ISSUE TITLE
Episode runner

### ISSUE THREAD

**Issue by deligentfool** [TonghanWang/ROMA#21]
你好，请问有episode runner版本的ROMA吗，因为我看到RODE论文里应该是用到的是episode runner版本的。我自己在您这个代码基础上改的episode runner版本的表现好像有点问题。

谢谢！

**Comment by TonghanWang** [comment]
不好意思，我们没有这个版本的ROMA

**Comment by deligentfool** [comment]
那RODE论文中的结果RODE是通过episode runner运行得到的,ROMA是通过parallel runner得到的结果吗?

**Comment by TonghanWang** [comment]
Yes, they are. The reason is that PyMARL uses a parallel runner at the time ROMA is developed.

(PyMARL and SMAC have many versions in the last few years. Only the latest uses an episode runner.)

**Comment by fyqqyf** [comment]
你好，如果没有 episode runner 的话，应该如何得到论文中演示的 replay 呢？
